xfeltor/plotting.py

```diff
@@ -267,16 +281,22 @@ def animate_pcolormesh(
 
     # Add title and axis labels
     ax.set_title(variable)
+    if None not in (xmin, xmax):
```

Comment:
You can pass `None` to ax.set_xlim:

```Python
ax.set_xlim(None, 10)
```
 Passing `None` leaves the limit unchanged. You could remove the `if None not in (min, max)` then.